[INTS-122] [SIEMINT-137] DDS: Greenhouse: Crawler Integration v1.0.0 (#18655)
DataDog/integrations-core: greenhouse/README.md

# Greenhouse Integration For Datadog

## Overview

[Greenhouse][1] is a cloud-based hiring platform that helps companies manage their recruitment processes. It features job postings, application tracking, candidate communications, interview scheduling, offers, and bulk actions, all aimed at helping companies hire the right people efficiently.

This integration focuses on ingesting audit logs, which document important events and configuration changes, enabling teams to detect incidents and minimize their impact.

> By seamlessly collecting these logs and channeling them into Datadog, the Greenhouse integration allows for comprehensive analysis. Leveraging the built-in logs pipeline, these logs are parsed and enriched, enabling effortless search and analysis. The integration provides insight into activities on the Greenhouse platform through the out-of-the-box dashboards.

Reviewer: ```suggestion
The Greenhouse integration collects these logs and sends them to Datadog. These logs are parsed and enriched through the built-in logs pipeline, which enables search and analysis. The integration also provides insight into activities on the Greenhouse platform through the out-of-the-box dashboards.
```

## Setup

### Configuration

1. [Create an API key from the Greenhouse Platform](#greenhouse-configuration).
2. [Configure the Datadog endpoint to forward Greenhouse events as logs to Datadog](#greenhouse-integration-configuration).

#### Greenhouse Configuration

> Steps to create API key on Greenhouse Platform:

Reviewer: ```suggestion
Steps to create an API key on the Greenhouse Platform:
```

1. Log in to **[Greenhouse][2]** with your credentials.
2. Click **Configure** located in the top right corner.
3. Navigate to **Dev Center** in the left panel.
4. Select **API Credentials**.
5. Click **Create new API key**.
6. Choose **Harvest** for API type.
7. Select **Unlisted vendor** for Partner (if your name is not in the list).
8. Add a description for your API key.
9. Click **Manage permissions**.
10. Copy and paste the API key as it cannot be retrieved later.
11. Under **Manage permissions**, check "Audit Log V1".

> (**Note**: If this option is not available, contact Greenhouse support to enable the audit log feature.)

Reviewer: If you have a link for Greenhouse support, recommend adding here
